webapp/components/signup/signup_controller.jsx

```diff
@@ -79,14 +79,14 @@ export default class SignupController extends React.Component {
                             }
                         );
                     },
-                    () => {
+                    (e) => {
                         this.setState({ // eslint-disable-line react/no-did-mount-set-state
                             noOpenServerError: true,
                             loading: false,
                             serverError: (
                                 <FormattedMessage
-                                    id='signup_user_completed.invalid_invite'
-                                    defaultMessage='The invite link was invalid.  Please speak with your Administrator to receive an invitation.'
+                                    id={e.id}
+                                    defaultMessage={e.message}
```

Comment:
I thought server will return the translated message. And client is just displaying the returned message.